bolt/lib/Target/AArch64/AArch64MCPlusBuilder.cpp

```diff
@@ -1582,6 +1607,12 @@ class AArch64MCPlusBuilder : public MCPlusBuilder {
     return Instrs;
   }
 
+  InstructionListType createDummyReturnFunction(MCContext *Ctx) const override {
```

Comment:
True, and initially I was tempted to do this. But then I thought it might be better for other targets to be explicitly aware of this workaround that gets used by Instrumentation.

Another argument, is that it might let new targets omit such a test. This could have been true for AArch64 if it was handled already like that. To be a better example, I've improved the AArch64 in my latest patch.

These are my thoughts, and not an absolute opinion. What do you think?

Reply:
Done, thanks.